.github/workflows/build.yaml

```diff
@@ -283,15 +283,15 @@ jobs:
 
   # use the oldest python version we support for this build
   test-ancient-virtualenv:
-    name: "test on py3.7, using old virtualenv"
```

Comment:
Is this test still relevant? [virtualenv 16.7.12's trove classifiers](https://pypi.org/project/virtualenv/16.7.12/) only claim to support Python 3.7 and lower (though its actual constraints are Python 2.7 and 3.4+).

My gut is that this job should be re-evaluated.

Reply:
It definitely needs to be re-evaluated.
I need to look at the commit history and check why we have this test. I remember that it was something to do with the `site` module being broken on old venvs, but it might not be relevant anymore.

I was going to check and maybe remove it after this PR, but we could fold it in here.

Reply:
It doesn't have to be part of this PR, but I wanted to get initial feedback for this comment before marking this as "LGTM". :grinning: